tests/flow/test_msf.py

```
@@ -0,0 +1,477 @@
+from common import *
+import random
+
+GRAPH_ID = "MSF"
+GRAPH_ID_RAND = "MSF_rand"
+
+class testMSF(FlowTestsBase):
```

Ruff (0.11.9)

`FlowTestsBase` may be undefined, or defined from star imports

(F405)